openapi/kas-fleet-manager.yaml

```diff
@@ -1789,8 +1794,12 @@ components:
         - cluster_external_id
         - cluster_ingress_dns_name
         - kafka_machine_pool_node_count
+        - access_kafkas_via_private_network
       type: object
       properties:
+        access_kafkas_via_private_network:
```

Comment:
@pawelpaszki @pb82 this is the field that's going to be exposed to the user. Let me know what you think of the naming.

/cc @dimakis this might be another information that's need to be supplied by the user and sent to KFM when a cluster is being registered.

Reply:
@machi1990 is it just this value that needs to be returned? does this effect the CLI in any other way? 🤔

Reply:
@dimakis once this is in, it is a value that will need to be asked to the user, just like the machine pool node count and then passed in as a request payload field when registering an enterprise cluster in KFM. 

Additionally, this value will be returned in the clusters list endpoint, but you do not need to worry about that.

Reply:
That's perfect, @machi1990, thank you for the clarification. I'll see the changes in the sdk once it goes into kfm anyway. I'll know to include it then! 👍

Reply:
Thanks.

Reply:
@machi1990 - I think the name is fine